google-cloud-bigquery/LOGGING.md

```diff
@@ -4,7 +4,7 @@ To enable logging for this library, set the logger for the underlying [Google
 API
 Client](https://github.com/google/google-api-ruby-client/blob/master/README.md#logging)
 library. The logger that you set may be a Ruby stdlib
-[`Logger`](https://ruby-doc.org/stdlib-2.4.0/libdoc/logger/rdoc/Logger.html) as
+[`Logger`](https://ruby-doc.org/stdlib-2.5.0/libdoc/logger/rdoc/Logger.html) as
```

Comment:
Maybe we should just use `https://ruby-doc.org/stdlib/libdoc/logger/rdoc/Logger.html` which will redirect to the latest stdlib version.

Reply:
Done, thanks!